nav2_smart_planner/CMakeLists.txt

```diff
@@ -23,36 +23,37 @@ include_directories(
   include
 )
 
-set(executable_name dijkstra_planner)
-
-add_executable(${executable_name}
-  src/main.cpp
-)
-
-set(library_name ${executable_name}_core)
-
-add_library(${library_name}
-  src/dijkstra_planner.cpp
-  src/navfn.cpp
-)
+set(library_name smart_planner)
+set(executable_name ${library_name}_node)
```

Comment:
We've been making the library name be the special one and have been using:

set(library_name ${executable_name}_core)

I'd like to be consistent with this. 

The executable name in this case would just be smart_planner.

Reply:
See comment above, this is against ROS convention. I will open a different PR to change all the other libraries/executables over.